server/index.js

```
@@ -307,6 +307,78 @@ app.post('/api/cards/:id/review', async (req, res) => {
+  try {
+    // 1. Recuperamos la tarjeta existente
+    const selectRes = await queryD1('SELECT * FROM cards WHERE id = ?', [id]);
+    const card = getFirstResult(selectRes);
+    if (!card) {
+      return res.status(404).send('Tarjeta no encontrada');
+    }
+
+    // 2. Determinamos si estamos en modo test
+    const isTest = process.env.NODE_ENV === 'test' ||
+                   (process.argv[1] && process.argv[1].includes('jest'));
+    let audio_url = card.audio_url;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)